src/components/Bridge/utils.ts

```diff
@@ -0,0 +1,124 @@
+import { EvmChain, GasToken } from '@axelar-network/axelarjs-sdk'
+import {
+  AssetId,
+  avalancheChainId,
+  ChainId,
+  ethChainId,
+  fromAssetId,
+  toAssetId,
+} from '@shapeshiftoss/caip'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import { getAxelarQuerySdk } from 'components/Bridge/axelarQuerySdkSingleton'
+import { AXELAR_CHAIN_NAMES, AxelarChainName, BridgeAsset } from 'components/Bridge/types'
+
+export const wrapAxelarAssetIdFromEthereumToAvalanche = (assetId: AssetId): AssetId | undefined => {
+  const chainId = KnownChainIds.AvalancheMainnet
+  const assetNamespace = 'erc20'
+  switch (assetId) {
+    // USDC on Ethereum
+    case 'eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48':
+      // axlUSDC on Avalanche
+      return toAssetId({
+        chainId,
+        assetNamespace,
+        assetReference: '0xfab550568c688d5d8a52c7d794cb93edc26ec0ec',
+      })
+    default:
+      return undefined
+  }
+}
+
+export const unwrapAxelarAssetIdFromAvalancheToEthereum = (
+  assetId: AssetId,
+): AssetId | undefined => {
+  const chainId = KnownChainIds.EthereumMainnet
+  const assetNamespace = 'erc20'
+  switch (assetId) {
+    // axlUSDC on Avalanche
+    case 'eip155:43114/erc20:0xfab550568c688d5d8a52c7d794cb93edc26ec0ec':
+      // USDC on Ethereum
+      return toAssetId({
+        chainId,
+        assetNamespace,
+        assetReference: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+      })
+    default:
+      return undefined
+  }
+}
+
+export const getBridgeDestinationAsset = (assetId: AssetId): AssetId | undefined => {
+  const { chainId } = fromAssetId(assetId)
+  switch (chainId) {
+    case ethChainId:
+      return wrapAxelarAssetIdFromEthereumToAvalanche(assetId)
+    case avalancheChainId:
+      return unwrapAxelarAssetIdFromAvalancheToEthereum(assetId)
+    default:
+      return undefined
+  }
+}
+
+export const chainNameToAxelarEvmChain = (name: string): EvmChain => {
+  switch (name) {
+    case AXELAR_CHAIN_NAMES.Ethereum:
+      return EvmChain.ETHEREUM
+    case AXELAR_CHAIN_NAMES.Avalanche:
+      return EvmChain.AVALANCHE
+    default:
+      throw new Error(`chainNameToAxelarEvmChain: name ${name} not supported`)
+  }
```

Comment:
Better yet, improve the types so that `name` is properly types as a `AxelarChainName`.

Actioned in https://github.com/shapeshift/web/commit/33a3cf685cfa7c7039995340e4df120350e00393.